### ISSUE TITLE
The difference between Pairnorm and Batchnorm.

### ISSUE THREAD

**Issue by guyguygang** [LingxiaoShawn/PairNorm#3]
Hi,
  Really nice work! After reading your paper, I have a question about the difference between Pairnorm and Batchnorm, especially under the inductive setting. Could you please provide some insights? 

Thank you!

**Comment by LingxiaoShawn** [comment]
From math perspective BatchNorm is normalizing all node embeddings in a batch to be Gaussian(0,1) distribution by subtracting the row-wise mean and dividing the row-wise standard deviation. In contrast, PairNorm is doing pairwise distance normalization by subtracting mean and dividing the F norm of all nodes embeddings, for more intuitive explanation the figure in the paper shows the underlying mechanism. I haven't think much about inductive setting, batchnorm is designed for stablize the training but PairNorm is designed originally for solving oversmoothing problem which may not exists in inductive setting (such as graph classification).
